getGit.tex

```diff
@@ -91,51 +91,82 @@ \section{Introduction}
-    \begin{block}{Obtaining Neodymium:}
-        \vspace{0.15cm}
-        \circled{1}$\textemdash$bastn\"{a}site \\
-        \circled{2}$\textemdash${monazite} \\
-    \end{block}
-    \end{columns}
+    	\item{Git is a powerful version control system}
+    		\begin{itemize}
+    			\item{Git allows multiple people to easily update the same source}
+    			\item{Git makes keeping tracking of changes easy and robust}
+    		\end{itemize}
+    	\item{Git allows people around the world to collaborate on open and closed source projects}
+   	\end{itemize}
+    \vspace{1cm} % generate some space between title and content
+
+    % Keeping all of this for now as an example in case we need to do something like this
+    %\begin{table}[h]
+    %\centering
+    %\begin{tabular}{lcccc} \bottomrule[2pt]
+    %    Name & Symbol & $A_r$ & M.P. (K) & IE (J) \\ \bottomrule
+    %    Helium & He & $4.00$ & $1$ & $3.94e^{-18}$ \\
+    %    Carbon & C & $12.01$ & $773$ & $3.94e^{-18}$ \\
+    %    Arsenic & As & $74.92$ & $1090$ & $1.48e^{-18}$ \\
+    %    Gold & Au & $196.96$ & $1337$ & $1.48e^{-18}$ \\
+    %    Cobalt & Co & $58.93$ & $1495$ & $1.26e^{-18}$ \\
+    %\bottomrule[2pt]
+    %\end{tabular}
+    %\caption{Properties of Whoville Elements}
+    %\end{table}
+
+    %\vspace{-0.6cm} % compact spacing between table and text
+
+    %\begin{columns}[t]
+    %\column{4.5cm}
+    %\begin{block}{Trace rare earth metals:}
+    %\begin{itemize}
+    %    \item{Ytterbium}
+    %    \item{Neodymium}
+    %    \item{Praseodymium}
+    %\end{itemize}
+    %\end{block}
+    %\column{4.5cm}
+    %\begin{block}{Obtaining Neodymium:}
+    %    \vspace{0.15cm}
+    %    \circled{1}$\textemdash$bastn\"{a}site \\
+    %    \circled{2}$\textemdash${monazite} \\
+    %\end{block}
+    %\end{columns}
 
 \end{frame}
 
 \begin{frame}
     \frametitle{Git is not GitHub}
+    \begin{itemize}
+    	\item{Git is a version control system}
+    	\item{GitHub is a site cataloging some git repositories}
+    		\begin{itemize}
+    			\item{It is free to put open source repos on GitHub}
+    			\item{You have to pay to catalog closed source repos}
+    			\item{Not all git repos are catologed on GitHub}
+    			\item{Git repos absolutely do not need to be on GitHub}
+    		\end{itemize}
+    	\item{GitHub gives an easy way to access and manage git repos}
+    	\item{There are tons of open source repos on GitHub that you can go and check out right now}
+    \end{itemize}
     \vspace{1cm} % generate some space between title and content
 \end{frame}
 
 \begin{frame}
     \frametitle{Who uses git?}
-
+    \begin{itemize}
+    	\item{Everyone uses git}
+    		\begin{itemize}
+    			\item{Major corporations}
+    			\item{Independent software developers}
+    			\item{Students collaborating on projects}
+    			\item{Many MANY open source projects}
+    		\end{itemize}
+    	\item{Git can easily be used for open or closed source projects making it a diverse and powerful tool}
```

Comment:
@Oddant1 Please add governments to list of  user groups. https://government.github.com/community/ (No need to add link)

Next time we meet, let's discuss aesthetic for this slide. I have some ideas...

Reply:
Done, sounds good